website/pages/docs/agent/kv.mdx

```diff
@@ -49,7 +49,8 @@ Objects are opaque to Consul, meaning there are no restrictions on the type of
 object stored in a key/value entry. The main restriction on an object is size -
 the maximum is 512 KB. Due to the maximum object size and main use cases, you
 should not need extra storage; the general [sizing
-recommendations](/docs/commands/snapshot/restore) are usually sufficient.
+recommendations](https://www.consul.io/docs/agent/options#kv_max_value_size)
```

Comment:
Should this use a relative link so that a link checker can validate it properly?

I think `/docs/agent/options#kv_max_value_size` should work

Reply:
Fixed